posthog/models/filters/mixins/common.py

```diff
@@ -151,7 +151,7 @@ def breakdown_type(self) -> Optional[BreakdownType]:
     @cached_property
     def breakdown_group_type_index(self) -> Optional[int]:
         value = self._data.get(BREAKDOWN_GROUP_TYPE_INDEX, None)
-        if value and not (isinstance(value, int) and 0 <= value < GROUP_TYPES_LIMIT):
+        if value is not None and not (isinstance(value, int) and 0 <= value < GROUP_TYPES_LIMIT):
```

Comment:
this raises `TypeError` if value = '3', and no error if value = 8. This is wrong?

Reply:
Uh-oh, I should add a test for this :sweat_smile: